src/app-loader.tsx

```diff
@@ -111,34 +111,40 @@ export const AppLoader = ({ children }: { children: React.ReactElement }) => {
     let interval: any = null;
 
     const getNetworkLimits = async () => {
-      const [networkLimits, stats] = await Promise.all([
-        fetch(networkLimitsEndpoint).then((res) => res.json()),
-        fetch(statsEndpoint).then((res) => res.json()),
-      ]);
-
-      const restoreBlock = Number(
-        networkLimits.networkLimits.bootstrapBlockCount
-      );
-      const currentBlock = Number(stats.statistics.blockHeight);
-
-      if (currentBlock <= restoreBlock) {
-        appDispatch({
-          type: AppStateActionType.SET_BANNER_MESSAGE,
-          message: t("networkRestoring", { bootstrapBlockCount: restoreBlock }),
-        });
-
-        if (!interval) {
-          startPoll();
-        }
-      } else {
-        appDispatch({
-          type: AppStateActionType.SET_BANNER_MESSAGE,
-          message: "",
-        });
+      try {
+        const [networkLimits, stats] = await Promise.all([
+          fetch(networkLimitsEndpoint).then((res) => res.json()),
```

Comment:
I would prefer to avoid mixing promises + async await, makes it hard to read. Can we have a little utlility to do the fetch and `res.json` and call it with each endpoint?